docs/designers-developers/developers/data/data-core-block-editor.md

```diff
@@ -958,6 +958,16 @@ Returns an action object used in signalling that two blocks should be merged
  * firstBlockClientId: Client ID of the first block to merge.
  * secondBlockClientId: Client ID of the second block to merge.
 
+### __internalRemoveBlocksPure
```

Comment:
This naming convention is not standard.

Reply:
Normally for experimental functions we use __experimental I followed something similar and used __internal as a prefix, I was not aware we have a were following a standard for this. What you think would be the best prefix for a case like this?

Reply:
They're documented here:

https://github.com/WordPress/gutenberg/blob/535e0752051957f3e6ab7aca2e71e99ceffc9a6f/docs/contributors/coding-guidelines.md#experimental-apis

I raise as a concern of: Introducing new vocabulary contributes to lack of cohesiveness and understanding when not consciously agreed upon, documented, and used consistency.

For example, it's not clear to me what differentiates an "internal" and "unstable" function. The problem admittedly already exists to an extent between unstable and experimental, and its confusion has prompted more than a few conversations.

Reply:
> What you think would be the best prefix for a case like this?

I'd rather see the function not exist at all, if possible, per the other conversation thread. If needing to act before / after a dispatch takes effect is an issue, it's quite unlikely that this will be isolated, and this pattern does not seem like one which would scale well.